catalog/pipeline/fbc-release/0.2/fbc-release.yaml

```diff
@@ -36,8 +36,16 @@ spec:
     - name: addArches
       type: string
       description: List arches to be added to be built
+    - name: requestUpdateTimeout
```

Comment:
Do we need it as a Pipeline parameter? Probably yes, just wondering. If so, should we provide a healthy default?

Reply:
I did not think too deep about it. But I would say yes. Eventually something might break in the remote pipeline (or communication) and the answer never come. The default for the IIB -  IIRC - is 300 seconds. So I would add +30 seconds  just in case.

Reply:
added a task default timeout of 360 seconds.